raiden-ts/src/transfers/epics/utils.ts

```diff
@@ -108,3 +111,48 @@ export function matchWithdraw<
           'totalWithdraw' in data ? data.totalWithdraw : data.total_withdraw,
         )));
 }
+
+/**
+ * @param channel - Channel state
+ * @param config - Raiden config
+ * @returns FeeSchedule for channel
+ */
+export function getFeeScheduleForChannel(
+  channel: Pick<Channel, 'token'>,
```

Comment:
This is a helper function, and just declares by interface that this function only cares about a subset of the `Channel` type (for now). Passing a full `channel` is still fine, since larger types are assignable to subsets of them. This was done to express the narrowest requirement possible on what the helper utility's interface, and it helps to unit-test these functions by not having to generate big clumsy Channel and Config objects if this function only cares and uses a small subset of it.